\documentclass[letterpaper]{article}
%\documentclass[12pt,letterpaper]{article}
%\setlength{\textwidth}{480pt}
%\setlength{\textheight}{630pt}
%\setlength{\voffset}{0pt}

\usepackage{amsmath, geometry, graphicx}
\usepackage{natbib}
%\usepackage{float}
\usepackage{url,hyperref}
\usepackage{booktabs}
\bibliographystyle{plainnat}

% https://tex.stackexchange.com/questions/6758/how-can-i-create-a-bibliography-like-a-section
%\usepackage{etoolbox}
%\patchcmd{\thebibliography}{\section*}{\section}{}{}

\pagestyle{plain}

\title{Novel Drug Repurposing Opportunities for Alzheimer's Disease from
  Genomic Fine Mapping and Transcriptomic Disease Signatures}

  \author{Attila Jones, Evan Wu, Sudhir Varma, Tina(?), ..., Hae Kyung Im, Madhav Thambisetty}

\date{National Institute on Aging, NIH}

\begin{document}

\maketitle

\section{Introduction}

Late onset Alzheimer's disease (AD) is a neurodegenerative disorder and the most
common form of dementia affecting a growing number of individuals in ageing
societies for which effective treatments are
unavailable~\citep{Bondi2017,Masters2015}.  Repurposing drugs approved for other indications is
a promising approach  towards identification of effective disease-modifying AD
treatments~\citep{Pushpakom2019,Fang2021,Taubes2021}.  One of the key
milestones of the National Plan for Alzheimerâ€™s Disease is to ``Initiate
research programs for translational bioinformatics and network pharmacology to
support rational drug repositioning''~\citep{Aging}.

However, drug discovery in AD is hindered by our limited understanding of the
early etiological triggers of the disease, which potentially implicate a
multitude of cellular pathways and cell types, and which becomes gradually
confounded by a large number of compensatory and degenerative pathologies over
the decades-long disease progression~\citep{DeStrooper2016}.  In this work we
attempted to address this challenge in two ways.  Firstly, we derive plausible
mechanistic insights into AD from previous genome- and transcriptome- and
proteome-wide association studies (GWAS, TWAS,
PWAS)~\citep{Jansen2019,Kunkle2019,Gerring2020,Baird2021,Schwartzentruber2021,Wightman2021,Wingo2021}
comparing AD and control samples, including analyses of differentially
expressed genes expression (DEGs) in APOE3/3 versus APOE4/4 individuals as
well as from comparison of DEGs in iPSC-derived neurons, astrocytes and
microglia from APOE3/3 versus APOE4/4
individuals~\citep{Taubes2021,Lin2018}~(\ref{fig:workflow}A).

\begin{figure}
\hspace{0.05\textwidth}A\hspace{0.7\textwidth}B

\includegraphics[scale=0.85]{../../../figures/by-me/repurposing-study-desing/repurposing-study-design.pdf}
\includegraphics[scale=0.85]{../../../figures/by-me/drug-target-disgene-network/drug-target-disgene-network.pdf}
\caption{
  A) Workflow of computational drug repurposing to Alzheimer's disease (AD).
  B) Principle of network proximity-based drug screen for AD.  Drug 4 and 5 are
  likely efficient for AD because they target either directly an AD gene (drug
  4) or a gene interacting with AD genes (drug 5).  On the other hand, drugs
  1--3 only target genes at least three interactions away from AD genes and
  therefore are likely not efficient for AD.
}
\label{fig:workflow}
\end{figure}

Secondly, we exploit rich, mechanistic information represented by two
interconnected networks: (1) the network formed by all drugs and their target
genes (drug-target network) and (2) the human gene-gene interaction network
(known as the interactome), whose Alzheimer's disease
module~\citep{Barabasi2011} is defined by a set of AD genes,
Fig~\ref{fig:workflow}B.  Given these networks, we estimate the efficacy of
each drug for AD by evaluating network proximity~\citep{Guney2016}, which is
based on the drug's network topological distance from the AD module and thus
estimates the drug's effect on AD genes.  Network proximity was
shown~\citep{Cheng2018} to be a powerful, systems-level, alternative to wider
used, but merely correlative, approaches to computational drug repurposing
such as the connectivity map, CMap~\citep{Lamb2006}.  We present the results
and computational validation of our network proximity based drug screen as well as
experimental validation of three selected top ranking drugs.

\section{Results}

\subsection{Computational drug screen: evaluating network proximity}

Drugs with low network proximity score for any given set of disease genes were
previously shown to be good candidates for repurposing to that
disease~\citep{Cheng2018}.  We performed a network proximity based drug
repurposing screen for AD on 2413 approved or phase 3 drugs from ChEMBL
(Fig.~\ref{fig:workflow}A).  For each drug we computed eight values of network
proximity given eight sets of AD genes that we derived from eight
complementary information sources (one curated knowledge based source and
seven multi-omic sources, Table~\ref{tab:genesets}).  This resulted in eight
distinct lists of the same 2413 drugs ranked according to increasing network
proximity (Fig.~\ref{fig:workflow}) so that top ranking drugs could be
considered for repurposing.

\begin{table}
\footnotesize
\begin{tabular}{cll}
Name     & Description & Reference  \\
\hline
TWAS2+   & genes supported by $\ge 2$ AD-TWAS/PWAS & (TODO: Supp Table)  \\
knowledge& curated AD genes from the DISEASES database & \cite{PletscherFrankild2015} \\
agora2+  & genes supported by $\ge 2$ Agora studies & https://agora.adknowledgeportal.org \\
AD DE APOE3-APOE3 & AD vs control DEGs: APOE3/APOE3 background & \cite{Taubes2021} \\
AD DE APOE4-APOE4 & AD vs control DEGs: APOE4/APOE4 background & \cite{Taubes2021} \\
APOE3-4 DE neuron & APOE4 vs APOE3 DEGs: iPSC-derived neurons& \cite{Lin2018} \\
APOE3-4 DE astrocyte & APOE4 vs APOE3 DEGs: iPSC-derived astrocytes& \cite{Lin2018} \\
APOE3-4 DE microglia & APOE4 vs APOE3 DEGs: iPSC-derived microglia& \cite{Lin2018} \\
\end{tabular}
\caption{
AD gene sets used as inputs to drug repurposing screens of this study.
}
\label{tab:genesets}
\end{table}

Although the eight input AD gene sets had little overlap with each other
(Fig.~\ref{fig:screen}A), the resulting eight vectors of network proximity had
markedly high correlations (Fig.~\ref{fig:screen}B) so that the eight lists
of drug rankings shared substantial similarity.  This suggests that the
input AD gene sets, although hardly overlapping, are still close enough to
each other in the network's topology to result in similar drug rankings
(Fig.~\ref{fig:screen}C right), which helps validate our approach.

\begin{figure}
\hspace{0.05\textwidth}A\hspace{0.7\textwidth}B

\includegraphics[scale=0.5]{../../notebooks/2021-12-02-proximity-various-ADgenesets/named-figure/jaccard-input-AD-sets.pdf}
\includegraphics[scale=0.5]{../../notebooks/2021-12-02-proximity-various-ADgenesets/named-figure/corr-coef-input-AD-sets.pdf}

\hspace{0.05\textwidth}C\hspace{0.5\textwidth}D

\includegraphics[scale=0.5]{../../notebooks/2022-01-14-top-drugs/named-figure/separate-aggregate-ranks-heatmap.pdf}
\includegraphics[scale=0.5]{../../notebooks/2022-01-14-top-drugs/named-figure/separate-aggregate-ranks-heatmap-topk.pdf}

\caption{Scoring and ranking 2413 drugs based on their network proximity to
  various AD gene sets.  A) Eight AD gene sets were input to the workflow.
  Jaccard indices quantifying shared genes show that the sets are highly
  dissimilar to each other.  B) The resulting network proximity scores of all
  screened drugs, in contrast, are fairly similar across the different input
  AD gene sets.  C-D) The separate drug rankings across imput AD gene sets
  were aggregated into a single final ranking, which is color coded (yellow:
  top drugs, blue: bottom drugs).  Note that the area delimited by red
  rectangle in C) is expanded into D).
}
\label{fig:screen}
\end{figure}

Assuming our eight input AD gene sets were all equally relevant to Alzheimer's
disease, we aggregated the corresponding drug rankings giving uniform weight
to all gene sets.  This yielded a single final ranked list of the top 605
drugs (Fig.~\ref{fig:screen}C, TODO: supplementary table).  The very top
ranked ($\le 30$) drugs on the final list also tended to be top ranked on five
out of eight input rankings (Fig.~\ref{fig:screen}D,
Fig.~S\ref{fig:rank-diff}).

\subsection{Computational validation}

To validate our network based screen we asked to what extent the top ranked
drugs were enriched in drugs already investigated or approved for AD.  To that
end we examined the rediscovery rate for the top-$t$ drugs relative to that
for the bottom-$1808$ drugs (see Methods for definitions).
Fig.~\ref{fig:ad-drug-rediscovery} shows that rediscovery rate for the top-$t$
drugs was about $2-3$-fold relative to that for the bottom drugs depending on
the threshold $t$.  This finding validates our approach.  Interestingly several, but
not all, other common indications showed a similar enrichment pattern
(Fig.~S\ref{fig:ad-drug-rediscovery-multi}).

\begin{figure}
\includegraphics[scale=0.6]{../../notebooks/2022-01-14-top-drugs/named-figure/top-bottom-ratio-top-k.pdf}
\caption{
Computational validation: relative rediscovery rate of drugs already investigated or
approved for AD in the top-$t$ drugs ranked by network proximity.  The relative
rediscovery rate being $>1$ validates our network based screen.
}
\label{fig:ad-drug-rediscovery}
\end{figure}

We also checked the agreement between our network proximity based results
above and the results we obtained by using CMap with the same input AD gene
set.  Strikingly, we found no agreement between the two even when we
conditioned on the various cell types of the CMap data set
(Fig.~S\ref{fig:proxim-cmap},~S\ref{fig:proxim-cmap-celltype}).  Moreover,
different implementations of CMap yielded considerably different results
(Fig.~S\ref{fig:cmap-cmap}).  Given these conflicting observations and the
more realistic, systems-based, mechanistic modeling assumptions behind the
network approach, we omitted all results obtained with CMap from our study.

\subsection{Selected top ranking drugs}

We selected four top ranking compounds for further investigation
(Table~\ref{tab:selected-drugs}) based on evidence from \emph{in vivo} animal
experiments or cell based assays that suggest applicability to AD and other
neurodegenerative diseases (below).

Arundine, also known as 3,3-diindolylmethane, is the dimeric product of the
natural product indole-3-carbinol.  While Arundine has been mostly
investigated in the context of drug resistant tumors~\citep{Biersack2020}, its
closed analogs have been found to cross the blood-brain barrier and protect
mice from MPTP-induced neurotoxicity and
neurodegeneration~\citep{DeMiranda2013} by suppressing
gliosis~\citep{DeMiranda2014}, a key hallmark of AD~\citep{DeStrooper2016}.
Moreover, Arundine was shown to inhibit oxidative stress induced apoptosis in
hippocampal neuronal cells~\citep{Lee2019} and protected primary hippocampal
cell cultures from ischemia induced apoptosis and
autophagy~\citep{Rzemieniec2019}.  Interestingly, the latter effect was found
to depend on Arundine binding to the Aryl Hydrocarbon Receptor, product of the
AHR gene,~\citep{Rzemieniec2019} which was found to be a target of Arundine in
mouse but has not yet been confirmed in humans and consequently was not taken
into account in our human-centered network proximity based drug screen.
Therefore, the Arundine-AHR interaction represents an additional piece of
evidence for the potential utility of Arundine in AD treatment.

One of the Arundine target that our analysis did account for is the RGS4 gene
(regulator of G-protein signalling 4). RGS4 was found to be associated with
schizophrenia~\citep{Chowdari2002}, one of the best genetically characterized
neuropsychological disorder, which shares substantial heritability with
AD~\citep{Consortium2018} and in which, like in AD, autoimmune pathways have
been found to play significant role~\citep{Sekar2016a}.  RGS4 is notable
because it is also targeted by Chenodiol, another of our selected top-ranking
drugs.  Chenodiol, or anthropodeoxycholic acid, has been shown to be
neuroprotective in Huntington's disease~\citep{Keene2002}.  Moreover,
Chenodiol is a bile acid, and thus plausibly linked to
AD based on recent evidence by us and others~\citep{Varma2021,Baloni2020}.

Our third and fourth selected top drugs were Cysteamine and Cysteamine-HCl.
Cysteamine, whose targets also include RGS4, a derivative of the amino acid
cysteine.  Cysteamine can traverse the blood brain barrier, was approved for
cystinosis and eye diseases, and more recently it has been extensively investigated as a potential drug for neurodegenerative diseases, specifically Huntingon's and Parkinson's diseases due to its neuroprotective activity~\citep{Besouw2013,Paul2019}.  Previous evidence for Cysteamine's potential utility for AD includes its favorable effect on the impaired
cognitive abilities of APP-Psen1 mice~\citep{Cicchetti2019}.

\begin{table}
\footnotesize
\begin{tabular}{p{0.2\textwidth} | p{0.2\textwidth} p{0.2\textwidth} p{0.2\textwidth}}
%\begin{tabular}{l | p{3cm} p{3cm} p{3cm}}
\toprule
                                              Name &                           Arundine &              Chenodiol &                          Cysteamine \\
\midrule
                                  Alternative form &                                    &                        &                      Cysteamine-HCl \\
                                           Synonym &               3,3-Diindolylmethane &  Chenodeoxycholic acid &                            Cystagon \\
                                         ChEMBL ID &                       CHEMBL446452 &           CHEMBL240597 &                           CHEMBL602 \\
                              Approved indications &          cervical cancer, phase 3 & cholesterol gallstones &            cystinosis, eye diseases \\
\midrule
\multicolumn{4}{l}{\scshape Gene targets} \\
                                      %Gene targets &                                    &                        &                                     \\
                                     ChEMBL, human &                        RGS4, GPR84 &            RGS4, NTCP2 &                         RGS4, CP3A4 \\
%                        ChEMBL, human (alt. form) &                                    &                        & GEMI, LMNA, PLK1, BLM, NF2L2, TYDP1 \\
                                             Other &                                AHR &                        &                                     \\
\midrule
\multicolumn{4}{l}{\scshape AD-relevance, present study} \\
                             Rank among 2413 screened drugs &                                  4 &                     28 &                                  51 \\
\midrule
\multicolumn{4}{l}{\scshape AD-relevance, previous studies} \\
                                  In vivo evidence & \cite{DeMiranda2013,DeMiranda2014} &       \cite{Keene2002} &                \cite{Cicchetti2019} \\
                                 In vitro evidence &      \cite{Lee2019,Rzemieniec2019} &                        &          \cite{Besouw2013,Paul2019} \\
\bottomrule
\end{tabular}
\caption{
Top drugs resulted by the aggregation of multiple network proximity based
screens 
}
\label{tab:selected-drugs}
\end{table}

\subsection{Experimental validation}

We performed a series of AD-relevant cell based assays to investigate the
utility of three of the four selected drugs introduced above: Arundine,
Chenodiol and Cysteamine-HCl (Fig.~\ref{fig:cell-based-assays}; note that
Cysteamine-HCl is shortened to ``Cysteamine'' in the figure).  While none of
the drugs protected primary cultured neurons from Amyloid-$\beta$
(A$\beta$)-induced toxicity (MTT assay, Fig.~\ref{fig:cell-based-assays}A),
(A$\beta$) clearance in BV2 cells was improved by both Arundine and Chenodiol
but worsened by Cysteamine-HCl (Fig.~\ref{fig:cell-based-assays}B).
Fig.~\ref{fig:cell-based-assays}C shows that secretion of all three A$\beta$
species tested in H4 cells was significantly decreased by both Arundine and
Chenodiol, which further supports these drugs' potential for AD treatment
(Fig.~\ref{fig:cell-based-assays}C); Cysteamine-HCl, on the other hand,
decreased only one species (A$\beta$1-42).  Neither Tau phosphorylation,
measured as pTau231 level, nor total Tau was significantly changed
(Fig.~\ref{fig:cell-based-assays}D), which suggests that none of the selected
drugs appears to improve Tau-dependent AD pathophysiology (although note the
unexpected, significant, increase of pTau231:Tau ratio for Arundine).  The
same conclusion is supported by Fig.~\ref{fig:cell-based-assays}H showing
unchanged Tau aggregation in a cell-free assay. BV2 cells were protected from
lipopolysaccharide (LPS) induced neuroinflammation by Chenodiol as measured by
the MTT assay (Fig.~\ref{fig:cell-based-assays}E), while Arundine had no
effect and Cysteamine even increased cell death.  On the other hand, Arundine
was the only drug that protected cells from LPS-induced TNF-$\alpha$ increase
(Fig.~\ref{fig:cell-based-assays}E).  In the same neuroinflammation model,
interleukins (IL-10, IL-1b, IL-6) were not changed by any of the drugs except
for a weakly significant increase by Chenodiol
(Fig.~\ref{fig:cell-based-assays}E).  Only Arundine protected primary neurones
from any aspect of trophic factor withdrawal (PI cell death,
Fig.~\ref{fig:cell-based-assays}G); in contrast, Chenodiol and Cysteamine-HCl
actually promoted certain aspects of cell death (LDH, MTT,
PI~Fig.~\ref{fig:cell-based-assays}G).  Finally, none of the drugs impacted
various assays of neurogenesis and neurite outgrowth in primary cultured
neurons (Fig.~\ref{fig:cell-based-assays}F).

Taken these experimental results together, Arundine and Chenodiol, but not
Cysteamine-HCl, exhibited several AD-relevant neuroprotective properties.

\begin{figure}
\includegraphics[scale=0.5]{../../notebooks/2022-09-21-cell-based-assays/named-figure/cell-based-assays.pdf}
\caption{Cell-based assays: testing AD-relevant protective effects of three
  selected top drugs from the present computational screen: Chenodiol,
  Cysteamine-HCl (shortened to ``Cysteamine'' in this figure), and Arundine.
  TODO: ask Tina about details: Throughout panels A-H tested drug
  concentrations were $c_1 = ?$, $c_2 = ?$, $c_3 = ?$, $c_4 = ?$, $c_5 = ?$,
  $c_6 = ?$.  Statistically significant effects are marked with $\ast$,
  $\ast\ast$, $\ast\ast\ast$, and with increasingly deeper blue or red,
  denoting $p < ?$, $p < ?$,  $p < ?$, respectively.
  A) Amyloid-$\beta$ (A$\beta$)-induced toxicity combined with the MTT assay
  of cell viability in primary cultured neurons.
  B) A$\beta$ clearance in BV2 cells; improved clearance is indicated by
  increased A$\beta$ in lysate, decreased A$\beta$ in supernatant and
  decreased supernatant:lysate ratio.
  C) Secretion of three A$\beta$ species in H4 cells.
  D) pTau231 and total Tau level.
  E) Lipopolysaccharide (LPS) induced neuroinflammation in BV2 cells; MTT
  assay of cell viability and the following citokine levels: interleukins
  (IL-10, IL1b, IL-6), the chemokine KC/GRO, and tumor necrosis factor
  (TNF)-$\alpha$.
  F) Assays of neurogenesis and neurite outgrowth in primary cultured neurons.
  G) Assays of cell death, viability and apoptosis in response to trophic
  factor withdrawal in primary neurones.
  H) Tau aggregation (cell-free).
}
\label{fig:cell-based-assays}
\end{figure}


\section{Discussion}

Computational drug repurposing is a new, rapidly evolving field without
established standards and with a multitude of varied approaches applicable to
a growing, diverse multi-omic body of data and curated
knowledge~\citep{Pushpakom2019}.  During our present investigations two drug
repurposing studies on AD were published: \cite{Fang2021} nominated sildenafil
while~\cite{Taubes2021} bumetanide.  These two works differed both in their
definition of AD genes as well as in the approach used to score and rank
drugs: \cite{Taubes2021} analyzed APOE genotype-specific DEGs with
CMap~\citep{Lamb2006}, while \cite{Fang2021} used knowledge-based AD genes, as
well as DEGs without known APOE genotype, as inputs to a network proximity
based drug repurposing screen~\citep{Cheng2018}.  In this paper we show that,
while the input gene sets have considerable effect on network proximity based
drug scores, the latter still show significant correlation.  Surprisingly, we
also find that the network proximity approach taken by \cite{Fang2021} and
CMap used by \cite{Taubes2021} leads to fully discordant, uncorrelated
results.  Moreover, we find that various published implementation of CMap lead
to quite different, albeit still correlated, results.  These observations
caution about methodological details.

Although the present study shares several methodological limitations with
those of \cite{Fang2021,Taubes2021}, we may argue to have carried out the most
carefully designed drug repurposing screen to date on AD.  We based our screen
on a diverse collection of AD gene sets that included a curated knowledge
based gene set, meta-analyzed, aggregated gene sets from TWAS and PWAS as well
as DEG sets from transcriptomic studies controlling for APOE genotype and
other genetic factors~\citep{Lin2018}.  These varied AD gene sources are
expected to introduce markedly varying bias into our drug repurposing screen: for
instance, the TWAS based gene set is small as it only contains genes with
strong statistical evidence for causal involvement in AD but likely misses
many other causal genes, while the DEG sets are large and likely inflated with
many genes that are not causal to AD.  It is easy to see how combination of
such diverse gene sets prior to the screen may further worsen the associated
biases. To prevent that, we conducted multiple screens, each conditioned on a
gene set and performed \emph{a posteriori} rank aggregation.  The rank
aggregation strategy also revealed that even some non-overlapping AD gene sets
resulted lead to the discovery of highly overlapping sets of very top drugs
demonstrating the robustness of our workflow.  Interestingly, the non-neuronal
AD gene sets were exception of this tendency pointing at the value of further
characterization of non-neuronal cells in
AD~\citep{Lopes2022,Mathys2019,DeStrooper2016}.

TODO: Top drugs and selected top drugs.  Drugs nominated
by~\cite{Fang2021,Taubes2021}.

Our computational validation, based on the rediscovery of drugs previously
developed or repurposed for AD, revealed the enrichment of our top ranked
drugs for also some common non-AD indications.  Genetic correlations between
disease phenotypes~\citep{Consortium2018}, market-driven selection biases, and
other factors may underlie not only this observation but also the known
comorbidities of AD~\citep{Santiago2021}.  Such overlap between different
drugs' indications is also consistent with the known relationship between drug
repurposing and side effects~\citep{Ye2014}.  Further research is needed to
dissect relationships between genetic correlations, comorbidities, side
effects, protein and gene network topology, etc to be able to harness these and
integrate into drug repurposing techniques.  That, in combination with
expanding multi-celltype, multi-omic characterization of AD and its molecular
subtypes~\citep{Neff2021}, will herald a new generation of drugs for
Alzheimer's disease.

\section{Methods}

\subsection{AD gene sets}

Given an AD gene set, the network structure of the human interactome, and
another network formed by drugs and their targets (drug-target network), the
topological proximity of a given drug to the AD gene set can be
quantified\citep{Guney2016} and used as evidence for repurposing that drug for
AD.  Rather than compiling a single AD gene set from multiple studies, we
opted for evaluating the proximity of any drug separately for multiple AD gene
sets and aggregate the multiple proximity values subsequently (see
Discussion).  Table~\ref{tab:genesets} summarizes the AD gene sets used in
this study.  The genes contained in each set are listed in Table~S1.

\subsection{Previous and novel TWAS on AD}

We obtained AD gene sets from multiple TWAS.  Besides collecting
AD genes from previous transcriptome- and proteome-wide association studies
(TWAS, PWAS) we
conducted our own applying the S-MultiXcan approach~\citep{Barbeira2018} to
the two largest AD-GWAS to date~\citep{Schwartzentruber2021,Wightman2021}.
S-MultiXcan borrows statistical strength from multiple tissue types; we used a
predictive model of gene expression built on all tissues of the GTEx data set
or on only brain tissues.
%TODO: describe here the results of Haky et al:
%number of significant genes (check with Haky et al on FDR); find interesting
%novel genes based on their membership in AD-relevant pathways.

We conservatively took the top 30 genes and investigated their agreement with
top gene sets from seven previous TWAS and one previous PWAS (see Table~S2
listing genes contained in each set).  Hierarchical clustering of studies
based on shared genes (Fig.~S\ref{fig:twas-clustermap}) showed a fairly low
agreement for all pairs of studies.  The most similar study pair was our
present TWAS and the fine mapping of~\cite{Jansen2019} despite several
methodological differences between these two studies, which helps validate our
novel TWAS results.  As might be expected, the least similarity was found
between the only PWAS in our collection and all the other studies.

\subsection{Drug-target network}

We queried ChEMBL~\citep{Gaulton2017} (v29, July 2021) for phase 3 and 4 drugs
and those of their targets that satisfied the following conditions: (1) the
target is a human protein (2) its average $-\log_{10}\mathrm{Activity}$ or
$p\mathrm{Activity}$ is $\ge 5$, which corresponds to $\le 10 \mu M$
following~\cite{Cheng2018}.  Average $p\mathrm{Activity}$ activity is defined
as $\sum_{i=1}^n p\mathrm{Activity}_i \ge 5$, where $n$ is the number of
activity measurements for a given drug-protein pair and $\mathrm{Activity}_i$
is the standard value of the $i$-th measurement stored in the
activities.standard\_value SQL variable.  See
\href{https://github.com/attilagk/CTNS-notebook/blob/main/2021-10-24-chembl-query/drug_target_avg_activity.sql}{our SQLite script} for implementation.

\subsection{Gene-gene interaction network}

We used the network compiled by \cite{Cheng2019}, which consists 11133 nodes
(genes) and 217160 edges (gene-gene interactions).

\subsection{Network proximity calculations}

\cite{Guney2016} developed the network proximity method for in silico drug efficacy screening.  It
quantifies the topological proximity $d$ in the interactome of the set $T$ of a drug's target genes
to the set $S$ of disease genes: 

\begin{equation}
  d(S, T) = \frac{1}{|T|}\sum_{t \in T} \min_{s \in S} d(s, t)
\end{equation}
where $d(s,t)$ is the shortest path length between disease gene $s$ and drug
target $t$.  The equation can be interpreted as the average length of shortest
paths taken from all targets of a drug to all the disease genes.

We used \href{https://github.com/attilagk/guney_code}{our modified clone} of
the original, now obsolete, implementation of \cite{Guney2016}, which allows
running the code under Python~3.

\subsection{CMap analyses}

Since there are several implementations of CMap (Connectivity Map) analysis was carried out with several tools.  First, we used
\url{https://clue.io/} \citep{Lamb2006} and, for each drug, averaged
``norm\_cs'',
the normalized connectivity scores.  Second, we used L1000CDS$^2$ at
\url{https://maayanlab.cloud/L1000CDS2} and the averaged ``score'' variable.
Third, we used... TODO: ask Sudhir what implementation he used.

\subsection{Rank aggregation}

As Fig.~\ref{fig:workflow} shows, we aggregated 8 lists of network
proximity-ranked drugs into a single list with rank aggregation algorithms.
We performed rank aggregation with the TopKLists R package
%\citep{Schimek2015}
several times, each time with different algorithm.  Then we assessed those
algorithm's performance with Modified Kendall Distance and found the MC3
algorithm to perform the best (Fig.~S\ref{fig:kendall-dist}).  Finally, we
used the MC3-aggregated list in our workflow.

\bibliography{repurposing-ms}

\newpage
\section*{Supplementary Material}

\setcounter{table}{0}
\makeatletter 
\renewcommand{\figurename}{Table S} % nice
\makeatother

\setcounter{figure}{0}
\makeatletter 
\renewcommand{\figurename}{Figure S} % nice
\makeatother

\begin{figure}[p]
\includegraphics[scale=0.4]{../../notebooks/2022-01-14-top-drugs/named-figure/rank-diff.pdf}
\includegraphics[scale=0.4]{../../notebooks/2022-01-14-top-drugs/named-figure/rank-diff-100.pdf}
\caption{
Difference of drug rank between the final, aggregated list and the list
resulting from each network proximity based drug screen.
}
\label{fig:rank-diff}
\end{figure}

%\begin{figure}[p]
%\includegraphics[scale=0.28]{../../notebooks/2021-12-02-proximity-various-ADgenesets/named-figure/prox-z-scatterplot_matrix.png}
%\caption{
%Z score of 2413 drugs (points) based on various input AD gene sets (rows and
%columns).
%}
%\label{fig:prox-z-scatterplot-m}
%\end{figure}

\begin{figure}
\includegraphics[scale=0.6]{../../notebooks/2021-07-01-high-conf-ADgenes/named-figure/cluster-experiments-genes.pdf}
\caption{
Similarity of TWA/PWA studies on AD in terms of shared genes.
}
\label{fig:twas-clustermap}
\end{figure}

\begin{figure}[p]
\includegraphics[scale=0.6]{../../notebooks/2021-11-30-proximity-cmap/named-figure/cmap-proxim-apoe4apoe4.png}
\caption{
Drug repurposing scores obtained by network proximity approach and CMap
}
\label{fig:proxim-cmap}
\end{figure}

\begin{figure}[p]
\includegraphics[scale=0.4]{../../notebooks/2021-11-30-proximity-cmap/named-figure/cmap-proxim-celltype-apoe4-apoe4.png}
\caption{
Drug repurposing scores obtained by network proximity approach and CMap in
given the cell type of the CMap expression profiling.
}
\label{fig:proxim-cmap-celltype}
\end{figure}

\begin{figure}[p]
\includegraphics[scale=0.6]{../../notebooks/2021-10-04-CMap-discussion/named-figure/cmap-tools-consistency.pdf}
\caption{
Results under different implementations of CMap.
TODO: remove Sudhir's?
}
\label{fig:cmap-cmap}
\end{figure}

\begin{figure}[p]
\includegraphics[scale=0.4]{../../results/2022-01-14-rank-aggregation/kendall-distance.png}
\caption{
Several rank aggregation algorithms' performance assessed by modified Kendall
distance.
}
\label{fig:kendall-dist}
\end{figure}

\begin{figure}[p]
\includegraphics[scale=0.4]{../../notebooks/2022-01-14-top-drugs/named-figure/top-bottom-ratio-top-k-multi-1.pdf}
\caption{
Rediscovery of existing phase 1--4 drugs for AD and other indications.
}
\label{fig:ad-drug-rediscovery-multi}
\end{figure}

\end{document}
