\documentclass[letterpaper]{article}
%\documentclass[12pt,letterpaper]{article}
%\setlength{\textwidth}{480pt}
%\setlength{\textheight}{630pt}
%\setlength{\voffset}{0pt}

\usepackage{amsmath, geometry, graphicx}
\usepackage{natbib}
%\usepackage{float}
\usepackage{url}
\bibliographystyle{plainnat}

% https://tex.stackexchange.com/questions/6758/how-can-i-create-a-bibliography-like-a-section
%\usepackage{etoolbox}
%\patchcmd{\thebibliography}{\section*}{\section}{}{}

\pagestyle{plain}

\title{Novel Drug Repurposing Opportunities for Alzheimer's Disease from
  Genomic Fine Mapping and Transcriptomic Disease Signatures}

\author{Attila Jones, ..., Madhav Thambisetty}

\date{National Institute on Aging, NIH}

\begin{document}

\maketitle

Late onset Alzheimer's disease (AD) is a neurodegenerative disorder and the most
common form of dementia affecting a growing number of individuals in ageing
societies but effective treatment is still
missing~\citep{Bondi2017,Masters2015}.  This is, in great part,
due to our poor understanding of the disease's early molecular etiology,
which potentially involves a multitude of cellular pathways and cell types,
and which becomes gradually confounded by a large number of compensatory
and degenerative pathologies over the decades-long disease
progress~\citep{DeStrooper2016}.

Genome-wide association studies (GWAS) on AD discovered $\approx 40$ risk loci
to date~\citep{Jansen2019,Kunkle2019,Schwartzentruber2021,Wightman2021}
suggesting polygenic, or at least oligogenic~\citep{Zhang2020}, mechanism
involving 100 or even fewer causal genes.  Unfortunately, translating GWAS to
efficient AD drugs has been prevented so far by several obstacles: (1) the
identity of causal genes within risk loci is unclear except for a few like
APOE4 TODO: ref; (2) GWAS, in general, cannot detect all causal
genes~\citep{Cao2014}; (3) not all causal AD genes may be
druggable~\citep{Floris2018,Finan2017}.

Several transcriptome- or
proteome-wide association studies (TWAS/PWAS) on
AD~\citep{Jansen2019,Kunkle2019,Gerring2020,Baird2021,Schwartzentruber2021,Wingo2021}
tackled the first obstacle alone by combining Mendelian
randomization~\citep{DaveySmith2014,Lawlor2008}, imputation
techniques~\citep{Barbeira2018}, or Bayesian colocalization~\citep{Wen2017}
with large transcriptomic or proteomic data sets but the results of
these TWAS/PWAS are divergent owing to methodological heterogeneity.
Differentially expressed genes and their modules
second obstacle to translating GWAS

\bibliography{repurposing-ms}
\end{document}
